src/engine/strat_engine/filesystem.rs

```diff
@@ -153,6 +153,21 @@ pub fn create_fs(dev_path: &Path) -> EngineResult<()> {
     Ok(())
 }
 
+pub fn grow_fs(dev_path: &Path) -> EngineResult<()> {
```

Comment:
I agree, but that's a bigger issue. What does that mean for this PR? `grow_fs` matches the existing style of similar functions. Can we merge this and then fix it up with the rest of them in a separate PR?